packages/material-ui/src/OutlinedInput/NotchedOutline.js

```diff
@@ -50,6 +50,7 @@ export const styles = (theme) => {
         paddingLeft: 5,
         paddingRight: 5,
         display: 'inline-block',
+        whiteSpace: 'nowrap',
```

Comment:
Thanks for exploring this option, I hadn't thought about it. Unfortunately, I think that we should stick with the overflow hidden. While the *nowrap* solves the vertical scrollbar, it creates a horizontal on. Also, fundamentally, I think that the label and span should style should be as close as possible.

<img width="321" alt="Capture d’écran 2020-04-13 à 21 00 32" src="https://user-images.githubusercontent.com/3165635/79151027-06d96e00-7dca-11ea-83b8-3dfee59bfede.png">